packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -3568,17 +3546,72 @@ describe.skip('network stubbing', function () {
       .wait('@status2').its('response.statusCode').should('eq', 301)
     })
 
-    // https://github.com/cypress-io/cypress/issues/9549
-    it('should handle aborted requests', () => {
-      cy.intercept('https://jsonplaceholder.cypress.io/todos/1').as('xhr')
-      cy.visit('fixtures/xhr-abort.html')
-      cy.get('#btn').click()
-      cy.get('pre').contains('delectus') // response body renders to page
-      cy.wait('@xhr')
+    context('abort and cancel', () => {
+      // https://github.com/cypress-io/cypress/issues/9549
+      it('should handle aborted requests', () => {
+        cy.intercept('https://jsonplaceholder.cypress.io/todos/1').as('xhr')
+        cy.visit('fixtures/xhr-abort.html')
+        cy.get('#btn').click()
+        cy.get('pre').contains('delectus') // response body renders to page
+        cy.wait('@xhr')
+      })
+
+      it('stops waiting when an xhr request is canceled', () => {
+        cy.visit('http://localhost:3500/fixtures/generic.html')
+
+        cy.intercept('POST', /users/, {
+          body: { name: 'b' },
+          delay: 2000,
+        }).as('createUser')
+
+        cy.window()
+        .then((win) => {
+          const xhr = new win.XMLHttpRequest()
+
+          xhr.open('POST', '/users/')
+
+          xhr.send()
+
+          win.location.reload()
+
+          cy.wait('@createUser').its('state').should('eq', 'Errored')
+        })
+      })
+
+      it('stops waiting when an fetch request is canceled', () => {
+        cy.visit('http://localhost:3500/fixtures/generic.html')
+
+        cy.intercept('POST', /users/, {
+          body: { name: 'b' },
+          delay: 2000,
+        }).as('createUser')
+
+        cy.window()
+        .then((win) => {
+          const controller = new AbortController()
+          const { signal } = controller
+
+          fetch('/users/', { signal, method: 'POST' }).catch((e) => {
+          // do nothing on an abort
+          })
+
+          // if you abort too fast in firefox or safari, the fetch is never sent to the server for us to intercept
+          if (!Cypress.isBrowser('chrome')) {
```

Comment:
is this an issue in electron or edge?
```suggestion
          if (!Cypress.isBrowser({ family: 'chromium' })) {
```

Reply:
Hmm, yeah, `{ family: 'chromium' }` seems like it could be right. @mjhenkes thoughts?

Reply:
chromium should be fine here. I don't think this is an issue in edge or electron